routing.go

```diff
@@ -58,8 +58,7 @@ func MessageRespond(s *discordgo.Session, m *discordgo.MessageCreate) {
 		span.AddField("command", "source")
 		sendResponse(ctx, s, m.ChannelID, "You can find the source here: https://github.com/ChrisLGardner/go-discord-bot")
 	} else if strings.HasPrefix(m.Content, "featurerequest") {
-		span.AddField("command", "featurerequest")
-		sendResponse(ctx, s, m.ChannelID, "File your owned damned issue: https://github.com/ChrisLGardner/go-discord-bot/issues")
```

Comment:
I'm torn on where I'd want to handle sending the responses from. Most of the functions return the data and let the routing handle it which I think ends up being a neater pattern since it makes it easier to test the more complex functions without having to figure out how to mock the sendResponse function.

Reply:
I think leaving the response in the routing file makes sense, easier to follow the logic (imho)